src/ensembl/src/shared/image-button/ImageButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import defaultStyles from './ImageButton.scss';
+import ImageHolder from './ImageHolder';
+
+export enum ImageButtonStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  DISABLED = 'disabled',
+  DEFAULT = 'default',
+  HIGHLIGHTED = 'highlighted'
+}
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  description: string;
+  image: React.FunctionComponent<React.SVGProps<SVGSVGElement>> | string;
+  classNames?: any;
```

Comment:
Well, we know the type of this thing, don't we?

([example](https://www.typescriptlang.org/play/#src=export%20enum%20ImageButtonStatus%20%7B%0D%0A%20%20ACTIVE%20%3D%20'active'%2C%0D%0A%20%20INACTIVE%20%3D%20'inactive'%2C%0D%0A%20%20DISABLED%20%3D%20'disabled'%2C%0D%0A%20%20DEFAULT%20%3D%20'default'%2C%0D%0A%20%20HIGHLIGHTED%20%3D%20'highlighted'%0D%0A%7D%0D%0A%0D%0A%0D%0A%0D%0Aconst%20classNames%3A%20%7B%20%5Bkey%20in%20ImageButtonStatus%5D%3F%3A%20string%20%7D%20%3D%20%7B%0D%0A%20%20active%3A%20'foo-active'%2C%0D%0A%7D%0D%0A%0D%0Aconst%20invalidClassNames%3A%20%7B%20%5Bkey%20in%20ImageButtonStatus%5D%3F%3A%20string%20%7D%20%3D%20%7B%0D%0A%20%20active%3A%20'foo-active'%2C%0D%0A%20%20passive%3A%20'foo-passive'%0D%0A%7D))

Reply:
I've update it.